samples/wikipedia-android-advanced/auth/login.yml

```diff
@@ -1,4 +1,4 @@
-appId: org.wikimedia
+appId: org.wikipedia
```

Comment:
If we changed the app id. Wouldn't we need to change the app binaries?

Reply:
This was incorrect, it should be `org.wikipedia` and not `org.wikimedia`. The reason this was uncaught is that these are used as subflows so I'm mainly updating this for correctness